flowback/comment/views.py

```diff
@@ -4,11 +4,26 @@
 from rest_framework.views import APIView
 
 from flowback.comment.selectors import comment_list
-from flowback.comment.services import comment_create, comment_update, comment_delete
+from flowback.comment.services import comment_create, comment_delete, comment_update
 from flowback.common.pagination import LimitOffsetPagination, get_paginated_response
 from flowback.files.serializers import FileSerializer
 
 
+class CommentOutputSerializer(serializers.Serializer):
```

Comment:
CommentOutputSerializer should either be used within OutputSerializer of an APIView, alternatively if used in more than one place, put it in comment/serializers.py

Reply:
I have moved it to a new serializers.py file and reused it the same way